src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/resource/ExampleResource.kt

```diff
@@ -17,8 +24,54 @@ import java.time.LocalDateTime
 class ExampleResource(private val exampleApiService: ExampleApiService) {
 
   @GetMapping("/time")
+  @Tag(name = "Examples")
+  @Operation(
+    summary = "Retrieve today's date and time",
+    description = "This is an example endpoint that calls a service to return the current date and time. Requires role PRISONER_LOCATION_UI",
```

Comment:
might be worth swapping `PRISONER_LOCATION_UI` for the role required in the pre-authorise annotation to prevent confusion? same for the endpoint below

Reply:
oops copy and paste error 😁

Reply:
interesting the tests still pass even though I specified an incorrect security requirement.  Running the json through https://editor.swagger.io/ then fails though 🫨